provider/equinix/mocks/packngo.go

```diff
@@ -5,35 +5,36 @@
 package mocks
 
 import (
+	reflect "reflect"
+
 	gomock "github.com/golang/mock/gomock"
 	packngo "github.com/packethost/packngo"
-	reflect "reflect"
 )
 
-// MockDeviceService is a mock of DeviceService interface
+// MockDeviceService is a mock of DeviceService interface.
```

Comment:
I assume my tool-chain is different and this is adding the periods. Have you seen this before? Any hints?

Reply:
Or maybe I am just missing an auto-linter step?

Reply:
Probably not using the same version of gomock as we do. This change is fine.